toast_planck/reproc_ring.py

```diff
@@ -1955,6 +2006,20 @@ def build_templates(self, rings):
             self.mapsampler_freq = mapsampler_fg
             self.mapsampler_freq_has_dipole = False
 
+        # Load FSL beam mask and corresponding low resolution sky model (one for each detector)
+        if self.fslbeam_mask_path is not None:
+            if self.fslbeam_mask == None:
+                self.fslbeam_mask = {}
+                self.nside_fsl = {}
+                self.fsl_pixels = {}
+                if self.downgraded_mapsampler_freq is None:
+                    self.downgraded_mapsampler_freq = {}
+                for det in self.dets:
+                    self.fslbeam_mask[det] = hp.read_map(self.fslbeam_mask_path[det])
```

Comment:
I'm guessing this should be read in the nested ordering

Reply:
As long as the pixel number is translated into pointing angles using the appropriate pixel ordering, that will work. Since the pixel ordering varies in this source file, I would explicitly set `nest=False` here and again when translating the pixel number into angles.